utplsql-maven-plugin/src/main/java/org/utplsql/UtPLSQLMojo.java

```diff
@@ -11,94 +28,244 @@
 import org.utplsql.api.exception.SomeTestsFailedException;
 import org.utplsql.api.reporter.Reporter;
 import org.utplsql.api.reporter.ReporterFactory;
+import org.utplsql.helper.PluginDefault;
+import org.utplsql.helper.SQLScannerHelper;
+import org.utplsql.model.ReporterConfiguration;
+import org.utplsql.model.ReporterParameter;
 
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.SQLException;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+/**
+ * This class expose the {@link TestRunner} interface to Maven.
+ * 
+ * @author Alberto Hernández
+ *
+ */
+@Mojo(name = "test", defaultPhase = LifecyclePhase.TEST)
+public class UtPLSQLMojo extends AbstractMojo
+{
+	@Parameter(defaultValue = "jdbc:oracle:thin:@localhost:1521:ut3")
```

Comment:
I guess this is a leftover of your internal testing/development? I'd rather have it required than filled with some - most likely - wrong default value

Reply:
Hi Samuel,
These are the default parameters used by utplsql installer so we reuse them in order to add coherence to the tool.
It is not great effort to change them to use the same parametrization than Java API project.

Reply:
I think that the default value in this place will not be of much use in real-life scenarios.
At work I never use local DB, at home I use 3 different local DB versions using docker.
In both cases, I'd need to adjust that parameter.
Thinking of team collaboration, the db connection&credentials would probably be placed outside of project-specific pom anyway.

Reply:
Changed for the same parameters as the java-api